### PR TITLE
Travis build.sh not shown mistake correction - mock waiter

### PR DESCRIPTION
**Pull request by cibo94**

### CODE REVIEW

File: rel-eng/travis/build.sh [hunk 1]
```diff
@@ -30,13 +30,14 @@ echo -en "travis_fold:start:$package_basename-mock\\r"
...
-while ! kill -0 ${proc_id}; do
-	sleep 60
-	minutes=$((min+1))
-	echo "Mock is working ${minutes}"
+mock_pid=$!
+secs=0
+while ps -p $mock_pid > /dev/null; do
+	sleep 5
+	secs=$((secs+5))
+	printf "\n>>> Mock is working for ${secs}s <<<\n"
```
Comment (jsilhan) [inline rel-eng/travis/build.sh]: do you need to print it every 5 secs? Could you use "\r" to not print the new line in console with new mock check/

**Comment by cibo94** [comment]
yeah I was only trying if it is working but tito is failing

**Comment by cibo94** [comment]
We have big problem with this. For no reason Tito have started failing so tests shall not pass. Please do not merge any more pull requests, because tests, as you can see in Travis build log, mock didn't run at all.

**Comment by cibo94** [comment]
I don't know where problem is but this ain't work.

```bash
$ git checkout -qf 1543c195d7eb2b89988be6072d3bc212d3fd63b9
$ git describe --match=rpg-0.0.1-1 1543c195d7eb2b89988be6072d3bc212d3fd63b9
fatal: No names found, cannot describe anything.
```

**Pull request closed without merging**